### ISSUE TITLE
Wiki Pages are full of examples that no longer valid

### ISSUE THREAD

**Issue by ghfields** [tpm2-software/tpm2-tools#1864]
Due to remapping of options and removal of commands, the wiki pages are very hard to follow.
https://github.com/tpm2-software/tpm2-tools/wiki/How-to-use-tpm2-tools is a great example of this problem.

It would be really helpful if the commands and syntax were updated throughout the entire wiki for the tpm2-tools 4.x release.  It also may help to notate on all the pages the version of tpm2_tools that was used to write that wiki page, like was done on the above example.

**Comment by macpijan** [comment]
I have started updating it in this repo: https://github.com/macpijan/tpm2-tools-wiki/commits/master

I hope to have some more this week. Although I feel like I might have some difficulties with some of the commands as I'm not that proficient with the tools yet.

**Comment by osresearch** [comment]
Having more worked out examples would be very helpful.  The ones in the man pages are near useless - as an example of this, I'm trying to figure out how to do remote attestation and thanks to [this stackexchange answer](https://security.stackexchange.com/a/204271/235159), I learned that `tpm2_makecredential` is required in some way to query the TPM if it generated the attestation key. So let's see what the man page says:

```
tpm2_makecredential(1) - Load an object that is not a Protected Object into the TPM.
```

Ok... maybe the example section will show how to use the command for this sort of attestation process:
```
tpm2_makecredential -e <keyFile> -s <secFile> -n <hexString> -o <outFile>
```

That's it. That is the entire section with no explanatory text or suggestions of where to find more details.

There's a longer "[Getting started with remote attestation](https://tpm2-software.github.io/tpm2-tss/getting-started/2019/12/18/Remote-Attestation.html)" guide that seems like it would be more helpful, except that it doesn't have any commands to run to do any of the numerous steps shown in the diagrams. This makes it very challenging to map to the actual command line arguments necessary to implement the process.

**Comment by williamcroberts** [comment]
@idesai I thought you were deleting wiki pages that were no longer relevant? Our goal is to get all the information down into the manpages or documents on the https://tpm2-software.github.io/ site.

Yeah the manpage for make credential sucks, thats a bug: #2051 
I filed a bug for more completeness to the guide: https://github.com/tpm2-software/tpm2-software.github.io/issues/42
